app/src/test/java/com/karumi/kataloginlogoutkotlin/LogInLogOutKataTest.kt

```diff
@@ -2,10 +2,11 @@ package com.karumi.kataloginlogoutkotlin
 
 import arrow.core.left
 import arrow.core.right
-import com.nhaarman.mockito_kotlin.whenever
+import com.nhaarman.mockitokotlin2.whenever
 import junit.framework.TestCase.assertEquals
-import junit.framework.TestCase.assertTrue
 import junit.framework.TestCase.assertFalse
+import junit.framework.TestCase.assertTrue
+import kotlinx.coroutines.runBlocking
```

Comment:
Why `LogInLogOutKata` uses `runBlocking` if there aren't coroutines in that file.

Reply:
Right, I forgot about it when removing the async code from the use case, thanks toni